src/styles/main.scss

```diff
@@ -0,0 +1,105 @@
+@import "stars";
+
+$color--text: #060b35;
+$color--text--title: #616070;
+$width--card--image: 160px;
+$height--card--image: 134px;
+$width--product-cards: 200px;
+$border--thickness: 1px;
+$background--color--card: #fff;
+$height--button: 40px;
+$width--button: 166px;
+$color--button: #00acdc;
+
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  font-family: Roboto, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.card {
```

Comment:
add padding for all card sides and remove extra margins from children
`padding: 32px 16px 16px`